statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -626,6 +802,12 @@ def __init__(self, model):
         # Setup the cache
         self._cache = resettable_cache()
 
+        # Setup covariance matrix notes dictionary
+        if not hasattr(self, 'cov_kwds'):
+            self.cov_kwds = {}
+        self.cov_type = 'opg'
```

Comment:
We do, but the problem is that I can't use the inherited `LikelihoodModelResults` machinery for `cov_type` because if `cov_type` is set it wants to run the model through `get_robustcov_results`.

If you like, I could augment that function to allow for these covariance types, but since they're specific to state space models it seemed like the wrong place to put them.

Of the covariance types available in that function, I only have references supporting the use of `cov_white_simple`.